src/main/scala/org/constellation/primitives/Genesis.scala

```diff
@@ -154,3 +164,11 @@ object Genesis {
           .map(tcd => dao.transactionService.put(tcd, ConsensusStatus.Accepted))
     }.toList.sequence.void.unsafeRunSync()
 }
+
+object GenesisObservationWriter {
+
+  final val FILE_NAME: String = "genesisObservation"
+
+  def writeToDisk(genesisObservation: GenesisObservation, path: File): Try[File] =
```

Comment:
Can you it into IO and change thread pool into `unbounded` which is responsible for blocking I/O operations (like writing to disk)